common/services/user.js

```diff
@@ -43,9 +44,16 @@ function getAuthGroups(token) {
 function getAuthLocations(token) {
   const { authorities = [] } = decodeAccessToken(token)
 
-  return getAuthGroups(token).then(groups => {
+  return getAuthGroups(token).then(async groups => {
     if (authorities.includes('ROLE_PECS_SUPPLIER')) {
-      return getLocationsBySupplierId(groups[0])
+      try {
+        const supplierKey = groups[0].toLowerCase()
```

Comment:
does this need protecting do we always have an element in the array?

Reply:
No, we won't necessarily have one, it could return an empty array.

But in those instances it will be because a user doesn't belong to any groups and the `try/catch` will catch that error and then just return an empty array which is the fallback case.